ocs_ci/utility/utils.py

```
@@ -3312,6 +3328,71 @@ def ceph_health_recover(
+                issue_key = pattern
+
+            # Get max attempts from fix_dict (default: 2)
+            fix_max_attempts = fix_dict.get("max_recovery_attempts", 2)
+
+            # Get and increment recovery attempt counter (initialize if doesn't exist)
+            current_attempt = tracking_dict.get("recovery_attempts", 0) + 1
+            tracking_dict["recovery_attempts"] = current_attempt
+
+            log.info(
+                f"Ceph health issue '{issue_key}' detected "
+                f"(attempt {current_attempt}/{fix_max_attempts})"
```

Ruff (0.15.13)

[warning] Logging statement uses f-string

(G004)